lib/webdriverio.js

```diff
@@ -230,6 +234,14 @@ let WebdriverIO = function (args, modifier) {
             })
     }
 
+    function mkRequestHanlderDecorator (opts) {
```

Comment:
I don't understand, what you trying to say that - "Also there is a wording the name"?
Rename "mk" - to "make" or "create" ?